packages/core/test/dialog/dialogTests.tsx

```diff
@@ -64,6 +64,62 @@ describe("<Dialog>", () => {
         document.body.removeChild(container);
     });
 
+    describe("ally support", () => {
```

Comment:
wrong spelling of "a11y", and actually I'd rather we just spell out the word here

```suggestion
    describe("accessibility support", () => {
```